lib/core/repository/cms/attribute_builder.rb

```diff
@@ -12,23 +12,21 @@ def self.build(response)
       end
 
       def attributes
-        attributes = response.body
+        attrs = response.body
```

Comment:
@tomas-stefano it is abbreviated to _attrs_ as the next best alternative _attributes_ is already used as the method name. Due to how it's heavily used by other methods, it's best to keep the sentiment of _attributes_ even if it's in an abbreviated form. 

```
  30       def set_title_from_label(attributes)
  31         attributes['title'] = attributes['label']
  32       end
  33
  34       def set_body_from_content_block(attributes)
  35         attributes['body'] = BlockComposer.new(attributes['blocks']).to_html
  36       end
  37
  38       def set_description(attributes)
  39         attributes['description'] = attributes['meta_description']
  40       end
  41
  42       def set_categories(attributes)
  43         attributes['categories'] = attributes['category_names']
  44       end
```